test/powershell/Language/Parser/UsingModule.Tests.ps1

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "Using module" -Tag "CI" {
+    BeforeAll {
+        Push-Location $PSScriptRoot
+        New-Item tmp -ItemType Directory
```

Comment:
Can we use Pester TestDrive?

Reply:
I was unaware of that Pester feature, I will see if it will work here.